test/autoupdate.test.js

```diff
@@ -2,94 +2,95 @@ require('./init.js');
 var assert = require('assert');
 var ds;
 
-before(function () {
+before(function() {
   ds = getDataSource();
 });
 
-describe('MS SQL server connector', function () {
-  it('should auto migrate/update tables', function (done) {
-
+describe('MS SQL server connector', function() {
+  it('should auto migrate/update tables', function(done) {
+    /* eslint-disable camelcase */
```

Comment:
Yea `schema_v1` is OK to be changed but table name is tricky... We'll take care of that later...